joomla/joomla-cms
# [Plugins] Add filters to manifest files.

Pull Request for Issue # .

### Summary of Changes

This adds filter attribute to some fields in plugin manifest files.

### Testing Instructions

Code review: 
1) Check that filtered default value is identical (including data type) to fallback value inside plugin code.
2) Check that comparisons are valid (not type safe or, if type safe, not compared to different type).

### Documentation Changes Required
No.
